src/modules/party-external-rating/dto/get-party-external-ratings-response.dto.ts

```diff
@@ -1,7 +1,7 @@
 import { ApiResponseProperty } from '@nestjs/swagger';
 import { DateString } from '@ukef/helpers/date-string.type';
 
-export type GetPartyExternalRatingsResponse = GetPartyExternalRatingsResponseElement[];
+export type GetPartyExternalRatingsResponse = GetPartyExternalRatingsResponseItem[];
```

Comment:
Why the change from element to item?

Reply:
Personally I just felt it made all the (already very long) names slightly shorter and more readable, as well as (to me anyway) making it even clearer that these refer to items in Arrays. Happy to change it back though

Reply:
Either name is fine with me as long as we're consistent, I wanted to understand the reason. Thanks Oscar